### PR TITLE
Adding Bootstrap Switch

### PR DESCRIPTION
**Pull request by rhamilto**
New widget for binary form input (e.g., on/off).  

@andresgalante or @erundle, please review and merge.

### CODE REVIEW

File: tests-src/bootstrap-switch.html [hunk 1]
```diff
@@ -0,0 +1,227 @@
+---
+alert-message: ', <a href="http://getbootstrap.com" class="alert-link">http://getbootstrap.com</a>, and <a href="http://www.bootstrap-switch.org/">http://www.bootstrap-switch.org/</a>.'
+categories: [Widgets]
+layout: page
+title: Bootstrap Switch
```
Comment (erundle) [inline tests-src/bootstrap-switch.html:5]: Should we call this toggle switch?

**Comment by andresgalante** [comment]
Although it is a toggle switch, we are using a component name "Bootstrap switch". I think it is a good idea to keep the original name, so it's easy to understand where we pick it from.
Plus this is the way we've been doing things so far, for example date pickers, data tables, etc.

File: tests/bootstrap-switch.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+(function() {
+  var $confirm;
+
```
Comment (erundle) [inline tests/bootstrap-switch.js:3]: So I know you just copied this from the example page for the project itself but do want to create any JS in our dist code base to support some of these actions?

Comment (rhamilto) [inline tests/bootstrap-switch.js:3]: I don't think there is a need; these actions are just for the purposes of demo-ing the widget.

**Pull request merged**